pkg/executor/builder.go

```diff
@@ -61,6 +61,7 @@ import (
 	"github.com/pingcap/tidb/pkg/parser/ast"
 	"github.com/pingcap/tidb/pkg/parser/mysql"
 	"github.com/pingcap/tidb/pkg/parser/terror"
+	"github.com/pingcap/tidb/pkg/planner/core"
```

Comment:
This package has been imported on line 65

Reply:
> This package has been imported on line 65

deleted